about.md

```diff
@@ -1,91 +1,91 @@
-
-The system [polymake](https://polymake.org) is a standard tool for dealing with convex
-polytopes, polyhedral fans, tropical hypersurfaces,
-and related objects from combinatorics and
-geometry. It is designed as a hybrid written in C++ and Perl. A
-sophisticated rule based mechanism decides which low-level C++
-functions are to be called to satisfy user demands.
-Polymake is integrated into OSCAR via the [Polymake.jl](https://github.com/oscar-system/Polymake.jl) Julia package.
-
-[Singular](https://www.singular.uni-kl.de) is a well-established computer algebra system for polynomial
-computations, with particular emphasis on applications in algebraic
-geometry, commutative algebra, and singularity theory, and with two
-subsystems for non-commutative algebra,
-[PLURAL](https://www.singular.uni-kl.de/Manual/4-1-2/sing_423.htm#SEC463) and
-[LETTERPLACE](https://www.singular.uni-kl.de/Manual/4-1-2/sing_789.htm#SEC841).
-Singular has a kernel written in C++ and provides its own
-interpreted user language. 
-Singular is integrated into OSCAR via the [Singular.jl](https://github.com/oscar-system/Singular.jl) Julia package.
-
-
-#### 📙 For More Info See The [OSCAR Book](http://book.oscar-system.org)
-
-## The role of Julia
-
-[Julia](https://www.julialang.org) is a an actively developed
-interactive and expressive programming language. Julia offers:
-
-* data types that can depend on other types, for example modeling
-matrices over polynomial rings over number fields;
-* Just-in-time compilation (JIT) to compile code at runtime to retain
-efficient performance;
-* capabilities of accessing low-level C/C++ data structures in the
-kernels of the cornerstones efficiently;
-* easy access to standard third-party libraries, (e.g., for string
-manipulation, visualisation, networking).
-
-Julia serves as an integration layer allowing the four
-cornerstones to communicate in a more direct way than through
-unidirectional interfaces. Furthermore it serves as high-level
-language for implementing efficient algorithms utilizing all
-cornerstones.
+**OSCAR (Open Source Computer Algebra Resource)** is an open-source computer algebra system written in the [Julia programming language](https://julialang.org). It provides a unified, high-performance framework for computations in algebra, geometry, number theory, and polyhedral geometry. OSCAR builds on the capabilities of four major systems—[GAP](https://www.gap-system.org), [Singular](https://www.singular.uni-kl.de), [Polymake](https://polymake.org), and the now-retired [ANTIC](https://github.com/flintlib/antic), whose functionality lives on through [Hecke](https://github.com/thofma/Hecke.jl), [Nemo](https://github.com/Nemocas/Nemo.jl), and [AbstractAlgebra](https://github.com/Nemocas/AbstractAlgebra.jl). By combining these specialized tools under one umbrella, OSCAR enables workflows and mathematical functionality that go far beyond what each system offers on its own.
```

Comment:
While some people claim OSCAR stands for "Open Source Computer Algebra Resource", the website and our manual all universally state "Open Source Computer Algebra Research system" (hence `oscar-system.org`).